include/boost/geometry/srs/projections/proj/webmerc.hpp

```diff
@@ -0,0 +1,77 @@
+// Boost.Geometry
+
+// Copyright (c) 2022 Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_SRS_PROJECTIONS_PROJ_WEBMERC_HPP
+#define BOOST_GEOMETRY_SRS_PROJECTIONS_PROJ_WEBMERC_HPP
+
+#include <boost/geometry/srs/projections/impl/base_static.hpp>
+#include <boost/geometry/srs/projections/impl/base_dynamic.hpp>
+#include <boost/geometry/srs/projections/impl/factory_entry.hpp>
+#include <boost/geometry/srs/projections/impl/pj_msfn.hpp>
+#include <boost/geometry/srs/projections/impl/pj_param.hpp>
+#include <boost/geometry/srs/projections/impl/pj_phi2.hpp>
+#include <boost/geometry/srs/projections/impl/pj_tsfn.hpp>
+#include <boost/geometry/srs/projections/impl/projects.hpp>
+
+#include <boost/geometry/srs/projections/proj/merc.hpp>
```

Comment:
Note that if this file is included then the includes above and below are not needed.

But I'd propose to define this projection in the same file as merc projection. This is how it's done with projections sharing the same bases and this is how it's done in proj.

Reply:
See e.g. [sts.hpp](https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/srs/projections/proj/sts.hpp#L245-L263) where 4 projections are defined.